docs/source/devguides/deployment/Linux-Platform-Hardening-Recommendations-for-VOLTTRON-users.rst

```diff
@@ -1,4 +1,4 @@
-.. _Platform-Hardening-for-VOLTTRON:
+===============================
```

Comment:
This allows a link to this page from other places in the documentation and should not be removed.